source/index.txt

```diff
@@ -40,6 +40,59 @@ deployments hosted in the following environments:
 
 .. include:: /includes/fact-environments.rst
 
+Related Tools and Libraries
+---------------------------
+
+Object Document Mappers
+~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB and our partners provide several Object Document Mappers (ODMs) for Node.js that
+simplify database interactions and improve the developer experience. These ODMs include:
+
+- **Mongoose**, which helps enforce a semi-rigid schema at the application level. Mongoose
+  also provides features to assist with data modeling and general data
+  manipulation.
+- **Prisma**, which helps ensure data consistency by offering a type-safe database client
+  and an intuitive schema. 
+
+For more information about these ODMs, see the following resources:
+
+- :website:`MongoDB ORMs, ODMs, and Libraries </developer/products/mongodb/mongodb-orms-odms-libraries/>` article
+- `Mongoose <https://mongoosejs.com/>`__ documentation
+- `Prisma <https://www.prisma.io/docs>`__ documentation
+
+Packages
+~~~~~~~~
+
+You can install the following packages to expand the functionality of the {+driver-short+}:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 2
+
+   * - Component
+     - Description
+
+   * - `bson <https://www.npmjs.com/package/bson>`__
+     - BSON serialization and deserialization library used by the official
+       {+driver-long+}
+
+   * - `kerberos <https://www.npmjs.com/package/kerberos>`__
+     - C++ extension for Node.js that provides support for kerberos authentication
+
+   * - `mongodb-client-encryption <https://www.npmjs.com/package/mongodb-client-encryption>`__
+     - Library for client-side encryption functionality
+
+   * - `mongodb-legacy <https://www.npmjs.com/package/mongodb-legacy>`__
+     - Package that supports callbacks to ease the migration process to the Promise-only API
```

Comment:
S: sort of sounds like it's adding callback support to the driver; it's really a whole different driver that supports callbacks.

grabbed this language from the [what's new](https://www.mongodb.com/docs/drivers/node/current/whats-new/#what-s-new-in-4.10) page:

```suggestion
     - Legacy Node.js driver with optional callback support
```